build.rs

```diff
@@ -54,7 +54,7 @@ fn concat_paths(first: &str, second: &str) -> PathBuf {
 	path
 }
 
-fn setup_android(config: &mut gcc::Config) {
+fn setup_android(config: &mut gcc::Build) {
```

Comment:
afaik, `gcc` is depraceted. We should use `cc` crate now. Maybe lets have another pr for that :)